HotelProjectsUI/hotels-app/src/components/Reservation/ReservationIdInput.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Button from "react-bootstrap/Button";
+
+export default (props) => {
+  return (
+    <div>
+      <label>Please enter reservaion id:</label>
+      {"  "}
```

Comment:
Why do you need this? want a new line? Just `</br>`

Reply:
This makes some spacing between the label and the input field so they won't be "glued" to each other.